tester/src/test/java/com/graphicsfuzz/tester/Util.java

```diff
@@ -154,22 +154,30 @@ static void assertImagesSimilar(File first, File second) throws FileNotFoundExce
     // TODO: This has been made very generous, based on Swiftshader producing visually identical
     // images with fairly high associated histogram distances.  If we find that bugs are slipping
     // through we should revise this.
-    assertTrue(ImageUtil.compareHistograms(first, second) < 2000.0);
+    assertTrue(ImageUtil.compareHistograms(first, second) < 500.0);
```

Comment:
Perhaps we should make these use the new image differencing metric, or file an issue with a TODO here to revisit in due course.

Reply:
Done. Issue #285